src/modules/esl-share/core/esl-share-trigger.ts

```diff
@@ -1,37 +1,103 @@
+import {attr, boolAttr, jsonAttr} from '../../esl-utils/decorators';
+import {ESLTraversingQuery} from '../../esl-traversing-query/core';
 import {ESLTrigger} from '../../esl-trigger/core';
+import {ESLSharePopup} from './esl-share-popup';
 
-import type {ESLToggleableActionParams} from '../../esl-toggleable/core/esl-toggleable';
+import type {ESLToggleable, ESLToggleableActionParams} from '../../esl-toggleable/core/esl-toggleable';
+import type {ESLSharePopupActionParams} from './esl-share-popup';
 
 /**
  * ESLShareTrigger component
  * @author Dmytro Shovchko
  *
- * ESLShareTrigger is an extension of {@link ESLTrigger} that
- * - controls an internal popup of {@link ESLShare} module in `popup` rendering mode.
- * - forwards the sharing attributes from the parent share {@link ESLShare} component to its associated instance of {@link ESLPopup}
+ * ESLShareTrigger is a component that allows triggering {@link ESLSharePopup} instance state changes.
  */
 export class ESLShareTrigger extends ESLTrigger {
   public static override is = 'esl-share-trigger';
 
-  /** List of attributes to forward from the host to the target share {@link ESLPopup} */
-  public static forwardedAttrs = ['share-title', 'share-url'];
+  /**
+   * List of social networks or groups of them to display (all by default).
+   * The value - a string containing the names of the buttons or groups (specified with
+   * the prefix `group:`) separated by spaces.
+   * @example "facebook reddit group:default"
+   * */
+  @attr({defaultValue: 'all'}) public list: string;
+  /** Popup state marker */
+  @boolAttr() public popupShown: boolean;
```

Comment:
TODO: delete, replace with correct `this.isTargetActive` processing

Reply:
Removed as redundant